### DISCUSSION TITLE
Can defineEventHandler receive files?

### DISCUSSION THREAD

**Discussion by besscroft** [nuxt/nuxt#24237] [Questions] [answered]
Hello the community,

Can defineEventHandler receive files? I tried many methods but could not get the requested file.
Someone can point me to the good direction?

Thanks!

**Comment by MuhammadKhizar7** [comment, marked as answer]
How you are sending file should be in form data and receiving it on the server using `  const data = await readMultipartFormData(event);`

**Reply by besscroft** [reply]
Thank you for your answer. This method is effective. I wish you a happy life!
